exp.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "PocketGear",
+  "description": "PocketGear is a clean and simple Pokédex app for Pokémon GO you'll ever need, made by Pokémon enthusiasts to help you trainers get the best out of your Pokémons.",
+  "slug": "pocketgear",
+  "sdkVersion": "10.0.0",
+  "version": "1.0.0",
+  "orientation": "portrait",
+  "primaryColor": "#D85D61",
+  "iconUrl": "https://s3.amazonaws.com/exp-us-standard/icons/pocket-gear.png",
+  "loading": {
+    "iconUrl": "https://s3.amazonaws.com/exp-us-standard/icons/pocket-gear.png",
+    "hideExponentText": false
+  },
+  "androidStatusBarColor": "#000000",
+  "packagerOpts": {
+    "assetExts": [
+      "ttf", "otf"
+    ]
+  }
+}
```

Comment:
more context for other Exponent people who look at this

2. We need to have an iconUrl for iOS and for Android
3. We support primary color but accent color (default spinner color etc) is useful
4. I think this is a fairly new API, @satya164 could you try to get something upstream to React Native for this? As for Exponent, we should deprecate our existing android status bar config from exp.json